packages/lib-core/src/types/plugin.ts

```diff
@@ -1,16 +1,16 @@
 import type { Extension, LoadedExtension } from './extension';
 
-export type PluginMetadata = {
+export type PluginRuntimeMetadata = {
   name: string;
   version: string;
 };
 
-export type PluginManifest = PluginMetadata & {
+export type PluginRuntimeManifest = PluginRuntimeMetadata & {
```

Comment:
Note to self, maybe we can keep this as `PluginManifest`, can be changed later.

Reply:
Is there any other type of Manifest than the one we build for us to use at runtime? 🤔

Reply:
No, there shouldn't be any other type of manifest.

The build-generated vs. runtime manifest should have the same structure.

I'll revert this back to `PluginManifest`.